src/olympia/addons/tests/test_views.py

```diff
@@ -2247,6 +2248,9 @@ def tearDown(self):
         self.refresh()
 
     def perform_search(self, url, data=None, expected_status=200, **headers):
+        # Just so that the waffle switch db query is bypassed (it's cached).
+        waffle.switch_is_active('boost-webextensions-in-search')
```

Comment:
For a moment I wondered why… maybe it makes sense to say that it's so that `assertNumQueries` in the next line works properly. But maybe others will read things with more context than I do...